From a2beb9bda8ca5d058662733c60e75e43a8a4b8e2 Mon Sep 17 00:00:00 2001
From: Huang Rui <rui1.huang@intel.com>
Date: Tue, 23 May 2023 01:18:03 +0800
Subject: [FEATURE] update mesa version string

Signed-off-by: Huang Rui <rui1.huang@intel.com>
---
 VERSION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/VERSION b/VERSION
index 1f49317b75d..8718fb60092 100644
--- a/VERSION
+++ b/VERSION
@@ -1 +1 @@
-23.2.0-devel
+23.2.0-devel dg2-20230511
-- 
2.25.1

